### PR TITLE
todo app

### PR DESCRIPTION
**Pull request by zmzade**

### CODE REVIEW

**Comment by abdualrhman** [comment]
Hello Zahra,

Good job with homework!

There is some issues when you delete a todo from the list. When you delete a todo it delete all the similar todos from the list as well.

I left a comment in the code to how to fix it.

**Comment by abdualrhman** [comment]
Also, it's better practice not to have all your component in one file but rather separate them in different files.

**Comment by abdualrhman** [comment]
@zmzade Sorry I had the older version of your code. I can that you've fixed the issue 👍

File: react/week2/todo-list/src/App.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    return <NotItem text="No items" />;
+  }
+
+  return (
+    <ul>
+      {props.todorows.map((x, index) => {
+        return (
+          <TodoRow
+            key={index}
+            textDescription={x.description}
```
Comment (abdualrhman) [inline react/week2/todo-list/src/App.js]: The value of the `key` attribute should have a unique and stable value. The value of an item's index changes when list is updated. You can add the item's `id`

```suggestion
          <TodoRow
            key={x.id}
            textDescription={x.description}
```

Comment (zmzade) [reply react/week2/todo-list/src/App.js]: Thanks Abdualrhman for great comments 👍

**Pull request merged**